tools/rbp/setup-sdk.sh

```diff
@@ -23,9 +23,9 @@ fi
 
 if [ -d $XBMCPREFIX ]
 then
-  [ -O $XBMCPREFIX ] || SUDO="sudo"
+  [ -O $XBMCPREFIX ] || command -v sudo >/dev/null 2>&1 && SUDO="sudo"
```

Comment:
this will always set SUDO. You need to parenthesize it:
 [ -O $XBMCPREFIX ] || (command -v sudo >/dev/null 2>&1 && SUDO="sudo")